src/recipes.js

```diff
@@ -0,0 +1,11 @@
+const filterRecipes = (data, filterTerm) => {
+  let filteredRecipes = data.filter((recipe) => {
```

Comment:
We should default to `const` and not `let` to keep in line with industry standards, we're not reassigning this value so `const` should work

Reply:
true